### ISSUE TITLE
Unexpected word token: "if"

### ISSUE THREAD

**Issue by unicornware** [oguimbal/pg-mem#193]
# Describe the bug

It seems that [`DROP TYPE [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]`][1] is not implemented.

Because of this, I'm unable to use the library with [sequelize@7.0.0-alpha.6][2].

```
💔 Your query failed to parse.
This is most likely due to a SQL syntax error. However, you might also have hit a bug, or an unimplemented feature of pg-mem.
If this is the case, please file an issue at https://github.com/oguimbal/pg-mem along with a query that reproduces this syntax error.

👉 Failed query:

    DROP TYPE IF EXISTS "public"."enum_tokens_type";;

💀 Syntax error at line 1 col 11:

  DROP TYPE IF
            ^
Unexpected word token: "if". I did not expect any more input. Here is the state of my parse table:

    kw_index → %word ● 
    kw_sequence → %word ● 
    kw_function → %word ● 
```

# To Reproduce

```sql
CREATE TYPE "bug_status" AS ENUM ('new', 'open', 'closed');
DROP TYPE IF EXISTS "bug_status";
```

check error on https://oguimbal.github.io/pg-mem-playground

# pg-mem version

`v2.3.3`

# Additional notes

- `DROP TYPE  "bug_status"` also yields an error: `Unexpected word token: "\"bug_status\""`
- i'm aware of [query interception][3], just wanted to open up a new issue following my last one - #192

[1]: https://www.postgresql.org/docs/9.5/sql-droptype.html
[2]: https://github.com/sequelize/sequelize/tree/v7.0.0-alpha.6
[3]: https://github.com/oguimbal/pg-mem#intercept-queries

**Comment by oguimbal** [comment]
Yup, not implemented.

Will fix soon (that's an easy one).

Thanks for the report

**Comment by dgmike** [comment]
Hi,
I'm using last version of sequelize (6.19.0) and got same problem. Any resolution about it?

**Comment by oguimbal** [comment]
Hi, this should be fixed in `pg-mem@2.4.3`

Sorry for the delay
